packages/applications-service-api/src/controllers/documents.js

```diff
@@ -55,4 +55,53 @@ module.exports = {
       res.status(500).send(`Problem getting documents for project ${caseRef} \n ${e}`);
     }
   },
+
+  async getV2Documents(req, res) {
+    const { caseRef } = req.query;
+    const { pageNo = 1, searchTerm = null, filters = null } = req.body;
+
+    if (!caseRef) {
+      throw ApiError.badRequest('Required query parameter caseRef missing');
+    }
+
+    logger.debug(`Retrieving documents by case reference ${caseRef} ...`);
+    try {
+      const documents = await getDocumentsFromDocumentsApiService(
```

Comment:
Looks like we'll also need to call a different business service method since the current one still includes the filter for only stages 1, 2 and 3.  You need to only add the `order: [['date_published', 'DESC']],` within this new business service method too in order to preserve v1 functionality untouched.